src/metal/rt64_metal.cpp

```diff
@@ -1019,7 +1019,9 @@ namespace RT64 {
     }
 
     void MetalBuffer::unmap(uint32_t subresource, const RenderRange* writtenRange) {
-        // Do nothing.
+        if (mtl->storageMode() == MTL::StorageModeManaged) {
+            mtl->didModifyRange(NS::Range(writtenRange->begin, writtenRange->end - writtenRange->begin));
```

Comment:
Written range can be nullptr, this will crash otherwise. I think you might've been lucky that RT64 specifies it, but it won't always be the case.

Reply:
When it's nullptr, you'll want to specify the entire buffer (0, size) as the range instead.